indexer-app/src/cmr/indexer/data/elasticsearch.clj

```diff
@@ -51,6 +51,12 @@
          (:transaction-id concept)
          (map :transaction-id (:variable-associations concept))))
 
+(defmethod get-elastic-version :service
+ [concept]
+ (apply max
+        (:transaction-id concept)
+        (map :transaction-id (:service-associations concept))))
+
```

Comment:
So I do have a question related to this: do we need to change the `or` clause in the private function `get-elastic-id` below to also include a check for `:service`?

Reply:
We'd need to once we start indexing to an "all-revisions-index" for services. It looks like that will be next sprint with ticket https://bugs.earthdata.nasa.gov/browse/CMR-4657.

Reply:
Got it -- that makes sense. I've been running into the all-revisions-indexing code this whole week and was wondering what we were going to do about that ... ;-)